apps/web/src/components/layout/components/v2/BridgeUpdateModal.tsx

```diff
@@ -0,0 +1,123 @@
+  const [urlError, setUrlError] = useState('');
+  const [showBridgeUpdateModal, setShowBridgeUpdateModal] = useState(false);
+
+  const { environment, isLoading: isLoadingEnvironment, refetchEnvironment } = useEnvironment();
+  const location = useLocation();
+  useMemo(() => {
+    setInputUrl(getBridgeUrl(environment) ?? '');
+  }, [environment, location.pathname]);
+
+  const toggleBridgeUpdateModalShow = async () => {
+    await refetchEnvironment();
+    setShowBridgeUpdateModal((previous) => !previous);
+  };
+
+  const buildBridgeUpdateButtonTooltipText = (env: IEnvironment | undefined) => {
+    const bridgeUrl = getBridgeUrl(env);
+
+    if (!bridgeUrl) {
+      return 'No Novu endpoint defined!';
+    }
+
+    return (
+      <HStack>
+        <IconLink></IconLink>
+        {`Connected to ${bridgeUrl}`}
+      </HStack>
+    );
+  };
+
+  const updateButtonLabel = isLoadingEnvironment
+    ? 'Getting endpoint information...'
+    : buildBridgeUpdateButtonTooltipText(environment);
+
+  const onBridgeUrlChange = (event) => {
+    event.preventDefault();
+    setUrlError('');
+    setInputUrl(event.target.value);
+  };
+
+  const onUpdateClick = async () => {
+    setUrlError('');
+    setIsUpdating(true);
+    try {
+      new URL(inputUrl ?? '');
+      const result = await validateBridgeUrl({ bridgeUrl: inputUrl });
+      if (!result.isValid) {
+        throw new Error(`Tested URL isn't valid`);
+      }
+      await storeInProperLocation(inputUrl);
+      await refetchEnvironment();
+    } catch {
+      setUrlError('The provided URL is not valid and/or is not the Novu Endpoint URL');
+    }
+
+    setIsUpdating(false);
+  };
+
+  const storeInProperLocation = async (bridgeUrl: string) => {
+    if (isLocalEnv()) {
+      setTunnelUrl(bridgeUrl);
+    } else {
+      await updateBridgeUrl({ url: inputUrl }, environment?._id ?? '');
+    }
+  };
+
+  const isLoading = isLoadingEnvironment || isUpdating;
+
+  return (
+    <>
+      <Tooltip label={updateButtonLabel}>
+        <Button variant="filled" Icon={IconPencil} onClick={toggleBridgeUpdateModalShow}>
+          Update novu endpoint
+        </Button>
+      </Tooltip>
+      <Modal
+        opened={showBridgeUpdateModal}
+        title={<Title size={2}>Update novu endpoint URL</Title>}
+        onClose={toggleBridgeUpdateModalShow}
+      >
+        <Text color={colors.B15}>
+          NOTE: This action will change the source code of your workflow definitions. You will no longer see workflows
+          from the current endpoint.
+        </Text>
+        <Input
+          label={'Novu Endpoint URL'}
+          description={'Specify the full URL including the protocol and novu endpoint path'}
+          onChange={onBridgeUrlChange}
+          value={inputUrl}
+          disabled={isLoading}
+          error={urlError}
+        />
+        <HStack justify={'space-between'}>
+          <Flex align="center">
+            {/* TODO: how do I set the path of the docs? */}
```

Comment:
I'm not really sure how to configure the docs button to send me to the correct location

Reply:
@davidsoderberg Sounds like we could use your help here please!